npm_and_yarn/spec/dependabot/npm_and_yarn/file_parser/lockfile_parser_spec.rb

```diff
@@ -307,5 +307,20 @@
         )
       end
     end
+
+    context "for an npm7 workspace project with a direct dependency in the workspace" do
+      let(:dependency_files) { project_dependency_files("npm7/workspace_nested_package") }
```

Comment:
Should we add some more specs here for npm 7 lockfiles? e.g. not in a workspace

Reply:
Yeah maybe! I figured most of this was covered already in the `FileParser` specs, do you have any scenario's in mind?

Reply:
Yeah the added spec looks good 👍